tasks/css.js

```diff
@@ -72,7 +74,7 @@ gulp.task('scss:lint', () => gulp.src([`${pathBuilder.scssSrcDir}/**/*.scss`, ..
  * Uses our config rules to validate syntax and structure of the CSS.
  *
  */
-gulp.task('css:lint', () => gulp.src(`${pathBuilder.cssDistDir}/**/*.css`)
+gulp.task('css:lint', () => gulp.src([`${pathBuilder.cssDistDir}/**/*.css`, `-${pathBuilder.cssDistDir}/**/*.css.map)`])
```

Comment:
What does the addition to this line do? Exclude the map file?

Reply:
Yeh – it seems to include the map files as part of the stream without this and tries to minify them…

Reply:
Are you sure the minus operator works? The `!` is usually the operator to use to exclude files?

Reply:
I've just tested with this addition removed and the output was the same?

Reply:
Just tested and for some reason, it no longer errors.  Go figure (I swear it didn’t work the other day…!).

Have pushed the amend.